desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -103,18 +110,19 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
       else
         resolve()
 
+  # Lock the form- action.finally() callback executes when form submission is complete regardless of success
   loadingLock: ($element, action) ->
     return if $element.hasClass('is-loading')
     $element.addClass 'is-loading'
-    action().finally => $element.removeClass 'is-loading'
+    action().finally => $element.removeClass 'is-loading' unless @comboForm
```

Comment:
Right, my question is that in the case I'm talking about, the registration fails. Because it is a combo form i *think* the `finally()` code should yield control of the spinner to the bid form (bad), but instead it goes away (good). I'm worried it is some race condition that might not always be the case, leading to a locked loading button.